src/main/java/de/themorpheus/edu/taskservice/controller/solution/MultipleChoiceSolutionController.java

```diff
@@ -83,4 +79,15 @@ public void deleteAll(int taskId) {
 		return ControllerResult.of(dto);
 	}
 
+	public void deleteAll(int taskId) {
+		ControllerResult<SolutionModel> optionalSolution = this.solutionController.getOrCreateSolution(taskId, NAME_KEY);
+		if (optionalSolution.isResultNotPresent()) return;
+
+		List<MultipleChoiceSolutionModel> multipleChoiceSolutionModels = this.multipleChoiceSolutionRepository.findAllById(
```

Comment:
Same as in the other MR: Collection-usage is useless and we should use exist/count